packages/kit/src/core/preview/index.js

```diff
@@ -120,6 +118,9 @@ export async function preview({
 				render_handler();
 			}
 		} else {
+			if (initial_url.startsWith(config.kit.paths.base)) {
+				req.url = initial_url.slice(config.kit.paths.base.length);
+			}
```

Comment:
should the `assets_handler` call on the list line be inside this `if` block to be consistent with the `if (initial_url.startsWith(SVELTE_KIT_ASSETS))` just above this?

Reply:
Ah, moving the response into the if just makes it never respond when a resource isn't found, instead of responding with a 404 (because there just is no handler given). 404s are handled inside the handlers.